### ISSUE TITLE
Script doesn't continue after logging in

### ISSUE THREAD

**Issue by HeliosLHC** [makuto/redditLikedSavedImageDownloader#2]
Script will log me in (displays username:username) but fails to continue after that.
I have the correct settings (false for soft retrieve) and was able to remove the warning text (plain text authentication deprecation removal)
I was able to get the script to work once but I'm unsure what fixed it. Tested this on a different computer and the same issue arises. Thank you for creating this script :), It worked well the one time i managed to get it working (not sure what I did exactly lol)
Produced Text After Running the Script
`Username: USERNAME
C:\Python27\lib\site-packages\praw\decorators.py:77: DeprecationWarning: reddit intends to disable password-based authentication of API clients sometime in the near future. As a result this method will be removed in a future major version of PRAW.

For more information please see:

* Original reddit deprecation notice: https://www.reddit.com/comments/2ujhkr/

* Updated delayed deprecation notice: https://www.reddit.com/comments/37e2mv/

Pass ``disable_warning=True`` to ``login`` to disable this warning.
  warn(msg, DeprecationWarning)`

Using Python 2.7.12 with latest PRAW

**Comment by makuto** [comment]
In username:username the last username is your username?

**Comment by HeliosLHC** [comment]
I just replaced my username with a "generic" username, but yes it shows username:myusernamehere.

**Comment by makuto** [comment]
Alright, so what is the problem now? Is it not downloading images, or is there another error message?

The DeprecationWarning thing can be ignored. After the warning is printed, it'll take a couple minutes to retrieve your liked and saved posts, depending on how high your TOTAL_REQUESTS is

**Comment by makuto** [comment]
You should try lowering TOTAL_REQUESTS to something like 10. It should finish getting your Reddit posts and start downloading images in about a minute. If it's any more than 3-5 minutes, there's something wrong with the script.

**Comment by HeliosLHC** [comment]
Ah, never mind, you're right, it takes a while for it was gather images. With it set to 500 for requests it took around 7 minutes, yet oddly with 100 requests it took around 15 minutes to start. I just found it odd it took so long as it gave no log/python response during that period.
I just couldn't figure out how long I was supposed to wait (longer than expected).
Yes I fixed the warning by editing the scraper.py (I know it can be ignored). Thanks for the help, your script works great :)
